### ISSUE TITLE
Any model trained for other languages?

### ISSUE THREAD

**Issue by basque21** [zihangdai/xlnet#161]
Is it any XLNet model trained in other languages? 
Thanks!

**Comment by stefan-it** [comment]
I'm currenly running experiments for Basque with different BPE sizes 😅

**Comment by basque21** [comment]
joke?

**Comment by stefan-it** [comment]
No, I'm working on a project (together with @anebz) on contextualized embeddings for Basque. I've trained ELMo and Flair Embeddings from scratch (will be released in a few days) and XLNet embeddings are currently training. Evaluation will be done on EIEC for NER and Universal Dependencies for PoS tagging.

**Comment by basque21** [comment]
wow, sorry! I didn't think someone in Munich would be working with Basque 😅 In any case, I'm working mainly in Spanish (despite my nickname), so I was asking for some model in Spanish... but nice to know about you!
